src/Component.php

```diff
@@ -319,7 +319,7 @@ private function handleRawHtml( DOMNode $node, array $data ) {
 			$variableName = $node->getAttribute( 'v-html' );
```

Comment:
Maybe we should rename this too, since it’s now more general? How about `$expression` or `$htmlExpression`?

Reply:
sure - can do!